src/components/Property/Types/VectorProperty/ViewOptions/DefaultValueList.tsx

```diff
@@ -16,28 +13,29 @@ export function ValueList({
   const max = additionalData.MaximumValue;
   const step = additionalData.SteppingValue;
 
+  const isInt = step.every((s) => s >= 1);
```

Comment:
?

Reply:
😅 A bit of a hacky way to check if the property type is an int... I'll make a new type instead, similar to the ListProperty